dsl/src/main/kotlin/com/chutneytesting/kotlin/util/SystemEnvConfigurationParameters.kt

```diff
@@ -0,0 +1,38 @@
+package com.chutneytesting.kotlin.util
```

Comment:
I think we have to avoid to have a dependency on junit apart from **junit.engine** package.